Assets/Scripts/Combat/TargetSelector.cs

```diff
@@ -69,6 +75,15 @@ private void TargetAllEnemies()
         ReplyToCallingAbility();
     }
 
+    private void TargetAllAllies()
+    {
+        GameObject[] allies = CombatSystem.Combatants.
+          Where(combantant => combantant.CompareTag("Player")).ToArray();
```

Comment:
This tag might change the the future because of how the dialogue system works. Sield and Ganiel will have their own tags

Reply:
Just a comment to let you know

Reply:
Thanks for letting me know! Will change it if it does